d2l-input-time.html

```diff
@@ -0,0 +1,392 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="d2l-input-shared-styles.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown.html">
+<link rel="import" href="../d2l-dropdown/d2l-dropdown-content.html">
+<link rel="import" href="../iron-a11y-keys-behavior/iron-a11y-keys-behavior.html">
+<link rel="import" href="../iron-selector/iron-selector.html">
+<link rel="import" href="../d2l-typography/d2l-typography-shared-styles.html">
+<link rel="import" href="../d2l-intl-import/d2l-intl.html">
+
+<!--
+`d2l-input-time`
+Accessible, Localized Time Picker Input Element
+
+@demo demo/d2l-input-time.html
+-->
+
+<dom-module id="d2l-input-time">
+	<template strip-whitespace>
+		<style include="d2l-input-styles">
+			:host {
+				--d2l-input-line-height: 1;
+				--d2l-input-padding: 0.7rem 1rem;
+				--d2l-input-padding-focus: calc(0.7rem - 1px) calc(1rem - 1px);
+
+				display: inline-flex;
+				flex-direction: column;
+			}
+
+			input {
+				font-family: inherit;
+			}
+
+			.d2l-dropdown-opener {
+				max-width: 6rem;
+			}
+
+			d2l-dropdown-content {
+				--d2l-dropdown-verticaloffset: 10px;
```

Comment:
Looks like there is an option for this in the [d2l-dropdown-content-behavior](https://github.com/BrightspaceUI/dropdown/blob/master/d2l-dropdown-content-behavior.html#L286).  Are we able to use that?  I'm also wondering if we will see a pattern emerge for a smaller offset - this seems like something that we'll begin to see inconsistency on.

Reply:
The comment above the property has it marked as private, so I think the intention is to set this using the css variable as above.  The 20px default does seem very large and looks odd, but it probably makes more sense when there's some kind of connector or error like a menu has

Reply:
I think the `vertical-offset` option was added to support Navbars, since the dropdown would be appear too low given the specific aesthetic of those openers.  The default was intended to work with buttons and the openers in the dropdown collection, and for the most part, we wanted to keep things consistent unless the case was "special"... and so the property was commented as "private" to discourage that.

In doing some poking around, there appears to be some inconsistency that's cropped up already.  I suspect some of that could be the same - perhaps not 20px, but not all different either.  It would be good if we could reel that in... figure out the common thread there.

Reply:
Added https://github.com/BrightspaceUI/inputs/issues/25 to address if we don't move to select.